lib/typing/ext/time_test.go

```diff
@@ -15,7 +15,7 @@ func TestExtendedTime_MarshalJSON(t *testing.T) {
 		// Single value
 		bytes, err := json.Marshal(extTime)
 		assert.NoError(t, err)
-		assert.Equal(t, `"2025-09-13T00:00:00.123Z"`, string(bytes))
+		assert.Equal(t, `"2025-09-13T00:00:00.123456Z"`, string(bytes))
```

Comment:
re: my last comment - shouldn't this continue being formatted in ms since that's the format specified in `extTime`?